src/screens/Resources/ResourcesComponents/MultipleItemList.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Text } from 'react-native-paper';
+import { View, StyleSheet } from 'react-native';
+import Accordion from 'react-native-collapsible/Accordion';
+import PropTypes from 'prop-types';
+import { nav } from '../../../constants';
+
+
+export default function MultipleItemList({ snapshot, navigation, setLoading }) {
+  const [activeSections, setActiveSections] = useState([]);
+
+  function renderHeader(doc, _, isActive) {
+    return (
+      <View style={MultipleItemStyles.accordionHeader}>
+        <View style={MultipleItemStyles.accordionHeaderText}>
+          <Text style={MultipleItemStyles.resourceText}>
+            {doc.data.title}
+          </Text>
+        </View>
+        <View style={MultipleItemStyles.accordionHeaderIcon}>
+          <Icon name={isActive ? 'chevron-up' : 'chevron-down'} />
+        </View>
+      </View>
+    );
+  }
+
+  function renderContent(doc) {
+    return (
+      <View style={MultipleItemStyles.accordionContent}>
+        <Text>
+          {doc.data.description}
+        </Text>
+
+        <View style={MultipleItemStyles.moreButton}>
+          <Text
+            style={MultipleItemStyles.resourceText}
+            onPress={() => {
+              setLoading(true);
+              navigation.push(nav.item, { resourceID: doc.id });
+            }}
+          >
+            More&nbsp;
+            <Icon name="chevron-right" />
+          </Text>
+        </View>
+      </View>
+    );
+  }
+
+  function updateSections(currActive) {
+    setActiveSections(currActive);
+  }
+
+  return (
+    <Accordion
+      sectionContainerStyle={{ paddingBottom: 8 }}
+      sections={snapshot.docs.map((doc) => ({ data: doc.data(), id: doc.id }))}
+      activeSections={activeSections}
+      renderHeader={renderHeader}
+      renderContent={renderContent}
+      onChange={updateSections}
+    />
+  );
+}
+
+const MultipleItemStyles = StyleSheet.create({
```

Comment:
Also, it feels like a lot of styles are shared between MultipleItemStyles and SingleItemStyles, it would be great if we could reduce code reuse.

Reply:
I'll do this for now but I think the styles can be looked at more thoroughly in a future pr as well since i think this one is mostly a functionality-based one